### PR TITLE
Initial implementation

### PR DESCRIPTION
**Pull request by ezequielc**
## Description
- Used a class to better structure the code
- Added sinon for testing 
- Added a variable to flag test environment and not automatically initialize rise-init (borrowed the idea from common-template repo)
- Initial test to validate the infrastructure

## Motivation and Context
Playlist component epic

## How Has This Been Tested?
Build passes and validated locally

## Release Plan:
- As the Submitter, upon requesting review of this pull request, I confirm that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed. 
- As the Reviewer, upon approving the changes in this PR, I confirm I have reviewed and I agree that the [Release Checklist](https://help.risevision.com/hc/en-us/articles/360031119991) has been completed

#### Release Checklist Items Skipped?
If any Release Checklist items were intentionally skipped, please provide which ones and the reasons why

### CODE REVIEW

File: test/rise-init_test.html [hunk 1]
```diff
@@ -30,11 +32,32 @@
 
     <script type="module">
       suite('rise-init', () => {
+        const sandbox = sinon.createSandbox();
```
Comment (alex-deaconu) [inline test/rise-init_test.html:35]: @ezequielc should the sandbox not be created in a `setup()` block? Thanks.

**Review by alex-deaconu — APPROVED** [review]
@ezequielc looks good, one question inline. Thanks.

File: test/rise-init_test.html (hunk 1 above)
Comment (ezequielc) [reply test/rise-init_test.html:35]: @alex-deaconu I don't think so. This is how it is defined in other components as well: https://github.com/Rise-Vision/rise-data-weather/blob/master/test/unit/rise-data-weather.html#L45. Thanks!

Comment (alex-deaconu) [reply test/rise-init_test.html:35]: @ezequielc interesting, thanks for the clarification!

**Pull request merged**